tools/EMLassemblyline/make_eml.xml

```diff
@@ -34,7 +34,7 @@
             </param>
             <output name="metadataout" ftype="xml" count="1" >
                 <assert_contents>
-					<has_line line="<?xml version="1.0" encoding="UTF-8"?>" />
+					<has_size value="6700" delta="200" />

```

Comment:
Checking for the file content is usually better than checking for the size.
For XML in particular you have those nice XML assertions: https://docs.galaxyproject.org/en/latest/dev/schema.html#assertions-for-xml-data